SCClassLibrary/Common/Collections/SequenceableCollection.sc

```diff
@@ -1336,6 +1336,21 @@ SequenceableCollection : Collection {
 		}
 	}
 
+	unixCmdGetStdOut { arg maxLineLength=1024;
+		var pipe, lines, line, pid;
+
+		if(this.notEmpty) {
+			pipe = Pipe.argv(this, "r");
+			lines = "";
+			line = pipe.getLine(maxLineLength);
+			while({line.notNil}, {lines = lines ++ line ++ "\n"; line = pipe.getLine; });
+			pipe.close;
+			^lines;
```

Comment:
you could leave away the trailing semicolon. We are trying to do this where possible.